### PR TITLE
Various map fixes

### PR DESCRIPTION
**Pull request by thilan-tran**
<p align="center">

<p align="center">

## Changes
- fix map header positioning
- fix map buttons positioning
- dramatic map and markers speed improvement (switch to Apple instead of Google maps)
  - fixes light blue fill color??

## TODO
- visual glitch when switching to showing map nodes

### CODE REVIEW

File: src/containers/MapScreen.tsx [hunk 1]
```diff
@@ -183,7 +186,7 @@ const formatEventBotsData = (apiData: EventBot[]) => {
 			topRight: itemCount.toString() + ' items',
 			// TODO: fix distance, items sold, and bot image
 			bottomLeft: '0' + 'm away',
-			bottomRight: '0' + ' itemsSold',
```
Comment (dvorjackz) [inline src/containers/MapScreen.tsx:186]: Oops haha

File: src/components/MapMenuView.tsx [hunk 2]
```diff
@@ -14,6 +14,12 @@ import { Icon } from 'react-native-elements';
 
 import { ItemProps, HeaderProps, MapMenuProps } from '../types/inventoryTypes';
 
+const NAV_HEIGHT = 90;
```
Comment (dvorjackz) [inline src/components/MapMenuView.tsx:17]: Is this the height of the top bar with the logo? If the height is needed to be declared multiple times might be good to have a constants file

**Review by dvorjackz — APPROVED** [review]
Lgtm - before you merge maybe try a few colors for the pin colors to see if something contrasts a bit better with the light blue map

**Comment by zhang-edward** [comment]
Maybe we could try a drop shadow behind each pin? I'm suggesting this bc "light color with dark outline" is basically a universal fix for all contrast problems.

For example look at the instagram camera:
IMG_1FCA53D62226-1

**Review by zhang-edward — APPROVED** [review]

**Comment by thilan-tran** [comment]
<p align="center">


How's this? Could probably change the selected color as well, too much blue.

**Pull request merged**